In [1]:
name = "Jazzmin"
hourly_wage = 20.0
hours = 40
total = hourly_wage * hours
print(f" {name}  earns {total} per week")

 Jazzmin  earns 800.0 per week


In [2]:
temps = [61, 65, 72, 58, 70, 74, 68]
high = 0
for t in temps:
    if t > 65:
        high = high + 1
print (high)

4


In [3]:
def to_celsius(f):
    return (f-32) * 5/9

print(round(to_celsius(temps[0]), 1))

16.1


In [4]:
movie = {"title": "Frozen", "year": 2013, "rating": "PG"}
for key,  value in movie.items():
    print (key, ":", value) 

title : Frozen
year : 2013
rating : PG


In [5]:
import pandas as pd
df = pd.read_csv("chickwts.csv")
df.head()

,weight,feed
0,179,horsebean
1,160,horsebean
2,136,horsebean
3,227,horsebean
4,217,horsebean


In [6]:
df.shape

(71, 2)

In [7]:
df.describe()

,weight
count,71.000000
mean,261.309859
std,78.073700
min,108.000000
25%,204.500000
50%,258.000000
75%,323.500000
max,423.000000


The describe() output shows me that there are a total of 71 chickens, with an average weight of 261.3 and median of 258. The weights have a wide spread ranging from 108 and 423. I found it surprisng that the chickens weights varied a lot. 